internal/output/html/package_table_template.html

```diff
@@ -1,53 +1,62 @@
-    onclick="showVulnDetails('{{ $element.Name }}-{{ $rowIndex }}')">
-    <td>{{ $element.Name }}</td>
-    <td>{{ $element.Ecosystem }}</td>
-    <td>{{ $element.InstalledVersion }}</td>
-    {{ if ne $element.HTMLVulnCount.UnFixed 0 }}
-    {{ if eq $element.HTMLVulnCount.Fixed 0}}
-    <td class="warning">
-      {{ $element.FixedVersion }}
+  {{ $index := uniqueID }}
+  <tr class="table-tr package-tr {{if $element.HTMLPackageDetail }}has-layer-info"
+    data-layer="{{ $element.HTMLPackageDetail.LayerID }}{{ end }}" id="table-tr-{{ $index }}">
+    <td class="icon-td">
+      <div class="expand-icon" onclick="showPackageDetails('{{ $index }}')">
+        <i class="material-icons">play_arrow</i>
+      </div>
     </td>
-    {{ else }}
-    <td class="warning">
-      {{ $element.FixedVersion }} (fixes {{ $element.HTMLVulnCount.Fixed }}/{{ $element.HTMLVulnCount.Called }})
+    <td {{ if eq $element.HTMLVulnCount.Called 0 }}class="uncalled-text" {{ end }}>{{ $element.Name }}</td>
+    <td {{ if eq $element.HTMLVulnCount.Called 0 }}class="uncalled-text" {{ end }}>{{ $element.InstalledVersion }}</td>
+    <td>
+      {{ if ne $element.HTMLVulnCount.UnFixed 0 }}
+      {{ if eq $element.HTMLVulnCount.Fixed 0}}
+      <p class="fixable-tag no-fix">No fix available</p>
+      {{ else }}
+      <div class="tooltip">
+        <p class="fixable-tag has-fix">Partial fixes available</p>
+        <span class="tooltiptext">Upgrading the package will fix {{ $element.HTMLVulnCount.Fixed }} out of {{
+          $element.HTMLVulnCount.Called }} vulnerabilities</span>
+      </div>
+      {{ end }}
+      {{ else }}
+      <p class="fixable-tag has-fix">Fix available</p>
+      {{ end }}
     </td>
-    {{ end }}
-    {{ else }}
     <td>
-      {{ $element.FixedVersion }}
+      {{ if ne $element.HTMLVulnCount.Called 0 }}
+      {{ template "severity_summary_template.html" $element.HTMLVulnCount }}
+      {{ end }}
     </td>
-    {{ end }}
-    <td>{{ printSeverityCountShort $element.HTMLVulnCount }}</td>
   </tr>
   <tr>
     <td colspan="100%">
-      <div id="vuln-details-{{ $element.Name }}-{{ $rowIndex }}" class="vuln-details">
-        <p><strong>Source: {{ $element.Source }}</strong></p>
-        <p>Vulnerabilities:</p>
-        {{ range $index, $vuln := $element.CalledVulns }}
-        {{ if eq $vuln.Summary.FixedVersion "No fix available" }}
-        <p><a href="https://osv.dev/{{ $vuln.Summary.ID }}">{{ $vuln.Summary.ID }}</a>
-          ({{$vuln.Summary.SeverityRating}}) has no fix available</p>
-        {{ else }}
-        <p><a href="https://osv.dev/{{ $vuln.Summary.ID }}">{{ $vuln.Summary.ID }}</a>
-          ({{$vuln.Summary.SeverityRating}}) has a fixed version at {{$vuln.Summary.FixedVersion}}</p>
-        {{ end }}
+      <div id="table-tr-{{ $index }}-details" class="package-details">
+        {{ if and $element.HTMLPackageDetail (not (eq $element.HTMLPackageDetail.LayerID "")) }}
```

Comment:
Is this "Layer introduced in"? We should be clearer about exactly what Layer means here.